docs/content/latest/develop/ecosystem-integrations/wso2.md

```diff
@@ -0,0 +1,179 @@
+---
+title: WSO2 Identity Server
+linkTitle: WSO2 Identity Server
+description: WSO2 Identity Server
+aliases:
+menu:
+  latest:
+    identifier: wso2
+    parent: ecosystem-integrations
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use [WSO2 Identity Server](https://wso2.com/identity-and-access-management/) to manage access to YugabyteDB.
+
+## Prerequisites
+
+Before you can start using WSO2 Identity Server, ensure that you have WSO2 Identity Server installed in one of the following location, depending on your operating system:
+
+- Mac OS:  /`Library`/`WSO2`/`IdentityServer`/*`<IS_VERSION>`*/*`<IS_HOME>`*
+- Windows:  `C:\Program Files\WSO2\IdentityServer`\\*`<IS_HOME>`*  
+- Ubuntu:  /`usr`/`lib`/`wso2`/`wso2is`/*`<IS_HOME>`*
+- CentOS:  /`usr`/`lib64`/`IdentityServer`/*`<IS_HOME>`*
```

Comment:
Yes, it does look a bit weird, but we need to have IS_HOME in italics because it indicates that the user must change the values. 
Let's keep it as is for now.